config/manifests/bases/operand-deployment-lifecycle-manager.clusterserviceversion.yaml

```diff
@@ -123,4 +123,5 @@ spec:
   maturity: stable
   provider:
     name: IBM
+  replaces: operand-deployment-lifecycle-manager.v1.4.0
```

Comment:
Since we renamed the package to ibm-odlm, will this break upgrade from 3.5.x ?

Reply:
Since we uninstall ODLM from `openshift-operators`and re-install it in `ibm-common-servicecs` for upgrading from 3.5.x, It won't break the upgrade.